content/desktop/release-notes.md

```diff
@@ -21,6 +21,60 @@ Docker Desktop versions older than 6 months from the latest release are not avai
+- BuildKit now evaluates Dockerfile rules to inform you of potential issues.
+- Host networking support is now generally available.
+- You can now configure shell autocompletions in the Dashboard. Navigate to the **General** tab in **Settings**.
+- When authenticating in the CLI without Docker Desktop installed, you can now use a browser-based flow, eliminating the need for manual PAT generation.
+- containerd is the default image store for new Docker Desktop users. This can be disabled in **Settings**.
+
+### Upgrades
+
+- [Docker Compose v2.29.0](https://github.com/docker/compose/releases/tag/v2.29.0)
+- [containerd v1.7.19](https://github.com/containerd/containerd/releases/tag/v1.7.19)
+- [NVIDIA Container Toolkit v1.16.0](https://github.com/NVIDIA/nvidia-container-toolkit/releases/tag/v1.16.0)
+- [Docker Scout CLI v1.11.0](https://github.com/docker/scout-cli/releases/tag/v1.11.0)
+- [Kubernetes v1.30.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.30.2)
+- Linux kernel `v6.10`
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed an issue that caused containers started with `--net=host` and listening on an IPv6 address to be accessible from the host.
+- Improved the UX for enabling the containerd image store in the **Settings** tab.
+- Fixed an issue that caused a deadlock seen while using the `grpcfuse` filesharing option under heavy load.
+- Fixed a bug where Mac-specific admin settings were impacting other platforms.
+- The Kubernetes client now uses KIND.
+- IPv6 address blocks can now be specified in Docker Engine's `default-address-pools`.
+- Fixed an issue with the validation of the Docker Engine's `bip`, `fixed-cidr` and `fixed-cidr-v6`. Fixes  [docker/for-mac#7104](https://github.com/docker/for-mac/issues/7104).
+- Docker Engine's `default-network-opts` parameter is now properly validated.
+- VirtioFS performance improvements include increasing directory cache timeout, handling change notifications from the host, removing extra FUSE operations for security.capability attributes, optimizing host event detection, and providing an API to clean caches after container termination.
+- Docker Desktop now notifies when there is a port conflict in a host networking container.
+
+#### For Mac
+
+- The Configuration integrity check feature now provides more context around what has changed with your Docker Desktop configuration. For more information, see the [FAQs](faqs/macfaqs.md).
+- The Configuration integrity check feature shows an error when it fails to repair Docker Desktop.
+- Fixed a bug where the IPv6 TCP was set to `host.docker.internal`. Fixes [docker/for-mac#7332](https://github.com/docker/for-mac/issues/7332).
+- Fixed an issue where `docker-compose` symlink was pointing to an empty location. Fixes [docker/for-mac#7345](https://github.com/docker/for-mac/issues/7345).
+
+#### For Windows
+
+- Fixed an issue where some `wincred` values were being persisted after uninstall.
```

Comment:
The reporter needs to be credited here too! (like we've done for other bugs with security relevance outside of our threat model e.g https://docs.docker.com/desktop/release-notes/#for-windows-13 as example).

For this case:
```
(reported by Javier Yong ([@Javiery3889](https://github.com/Javiery3889))
```

Reply:
we don't credit individual users 🤔  do we? we link to issues to close them

Reply:
we have in the past. For things reported to security@ but outside of our threat model (so no CVE). c.c @gabriellavengeo to correct me if wrong

Reply:
Do we need to add more context that this was security related??

Reply:
ah true. The last one seems to have it's own Security section and then For Windows under that. I'm ok to have it structured that way here too